CMakeLists.txt

```diff
@@ -99,9 +99,23 @@ include_directories(
   "${LIB_INCLUDES}"
 )
 
+set(executables "")
+
+function(add_mvc_executable name)
+  set(executables
```

Comment:
Instead of `set`, should we use `list(APPEND, ` instead ? [link](https://cmake.org/cmake/help/latest/command/list.html#append)

That way, we could make the function accept a list of `name` or we could call it multiple times with different `name` values if we have more than one MVC later

Reply:
Yes, I've noticed that as well, but the list command does not support PARENT_SCOPE, which is why we cannot use it to modify a variable in the parent scope